python/lsst/ap/verify/ingestion.py

```diff
@@ -207,7 +215,15 @@ def _doIngest(self, repo, dataFiles, badFiles):
         badFiles : `list` of `str`
             A list of filenames to exclude from ingestion. Must not contain paths.
             May contain wildcards.
+
+        Raises
+        ------
+        RuntimeError
+            Raised if ``dataFiles`` is empty.
         """
+        if not dataFiles:
+            raise RuntimeError("No raw files to ingest.")
```

Comment:
Maybe be a little more explicit here, to help with debugging: `"No raw files to ingest; dataFiles is %s."%dataFiles`

Reply:
The `if` statement guarantees that won't display anything interesting.

Reply:
`dataFiles` could be `None`, `False`, `[]`, `()`, `{}`, `''`. It might be informative to know which of those it was.

Reply:
Ugh, weakly typed languages. 😞 I'll write it with `%r`, then.